### PR TITLE
Funding, Products, Partnerships - Atom Bank

### PR DESCRIPTION
**Pull request by rajalkothari**

### CODE REVIEW

File: data/account-providers/atombank.json [hunk 1]
```diff
@@ -60,5 +60,29 @@
       "url": "https://www.bbva.com/en/atom-bank-ceo-lead-challenger-banks-uk-open-banking-initiative/",
       "date": "2017-07-05"
     }
+  ],
+  "fundingTotal": {
+    "amount": "429000000",
+    "currency": "GBP"
+  },
+  "foundedYear": 2014,
+  "thirdPartyBankingLicense": false,
```
Comment (Gdewilde) [inline data/account-providers/atombank.json:69]: Should be empty string "" or null

File: data/account-providers/atombank.json [hunk 2]
```diff
@@ -60,5 +60,29 @@
...
+      "businessLoans": true,
+      "businessOverdraft": true
+    }
+  },
+  "partnerships": [
+    {
+      "name": "Thought Machine",
+      "icon": "https://d1nzzttg89jj6.cloudfront.net/assets/companies/1558349764/thoughtmachine-1558349764.png",
+      "websiteUrl": "https://www.thoughtmachine.net/",
+      "id": "thought-machine-uk",
```
Comment (Gdewilde) [inline data/account-providers/atombank.json:81]: id: thoughtmachine-net

File: data/account-providers/atombank.json [hunk 3]
```diff
@@ -60,5 +60,29 @@
...
+  "products": {
+    "loans": {
+      "businessLoans": true,
+      "businessOverdraft": true
+    }
+  },
+  "partnerships": [
+    {
+      "name": "Thought Machine",
+      "icon": "https://d1nzzttg89jj6.cloudfront.net/assets/companies/1558349764/thoughtmachine-1558349764.png",
```
Comment (Gdewilde) [inline data/account-providers/atombank.json:79]: Update url to https://res.cloudinary.com/apideck/icons/thoughtmachine-net.jpg

**Comment by rajalkothari** [comment]
File amended - Atom

**Pull request merged**